sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -771,6 +768,9 @@ export class CertificateClient {
 
     const generatedOptions: KeyVaultClientSetCertificateIssuerOptionalParams = {
       ...requestOptions,
+      credentials: requestOptions.credentials,
```

Comment:
isn't this redundant with the above? Like you're spreading all of `requestOptions`, but then re-applying the same key again here

Reply:
yes! My intention was to remove the ...requestOptions, but I left it by mistake. Considering how scoped this is, I'll keep the spread operator here only (and on similar instances).